src/net/udp.rs

```diff
@@ -161,6 +161,33 @@ impl UdpSocket {
         self.inner.local_addr()
     }
 
+    /// Returns the socket address of the remote peer this socket was connected to.
+    ///
+    /// # Examples
+    ///
+    ///
+    #[cfg_attr(all(feature = "os-poll", not(target_os = "freebsd")), doc = "```")]
```

Comment:
Why are you excluding FreeBSD?

Reply:
I copied the description for the `addr` function, I can remove this exception if it not necessary.

Reply:
I guess you copied from `local_addr`? The comment above it explains why, I don't think it's needed for `peer_addr`, so you can remove it.

Reply:
Yes.. I have the same doubts for other `cfg_attr` , I don-t have clear the objective for the `os-poll` feature